examples/library/test_manycore_parameters.h

```diff
@@ -49,12 +49,12 @@
 #define GLOBAL_INDICATOR_WIDTH 1
 #define GLOBAL_INDICATOR (1 << 30)
 #define GLOBAL_EPA_SIZE DMEM_EPA_SIZE
-#define GLOBAL_X_BITS 6
+#define GLOBAL_X_BITS 7
```

Comment:
Do we still run this test? Is this the one you hated with a passion?

Reply:
Yes. I do hate this test because all it really tests is that we updated the macros here.

Reply:
I'm fine with nuking this test

Reply:
Let's file an issue; the problem with this test is it duplicates code for materializing group/global EVAs. There's probably a better way to test this.